cf_client.rb

```diff
@@ -101,6 +101,21 @@ def create_space(name, organization_guid, developer_guids, manager_guids, space_
     }
     sr = @token.post("#{api_url}/spaces",
         body: req.to_json)
+    space = sr.parsed
+    space_guid = space["metadata"]["guid"]
+
+    self.add_space_asg(space_guid, "public_networks_egress")
+    self.add_space_asg(space_guid, "trusted_local_networks_egress")
+
+  end
+
+  def add_space_asg(space_guid, asg_name)
+
+    asg_response = @token.get("#{api_url}/security_groups?q=name:#{CGI.escape asg_name}")
+    asg = asg_response.parsed
+    asg_guid = asg["results"][0]["metadata"]["guid"]
+
+    bind_asg_response = @token.put("#{api_url}/security_groups/#{CGI.escape asg_guid}/spaces/#{CGI.escape space_guid}")
```

Comment:
Just checking that nothing needs to be done with the response from the `@token.put()` call here, correct?

It'll be returned, but I see that where this method is called doesn't do anything with the return value either; that's fine if nothing needs to be done because it's taken care oof with the API call itself, I just wanted to be sure!

Reply:
Yes, the result for `create_space` is never used in the `monitor.rb` script.